Update sql-data-warehouse-reference-powershell-cmdlets.md

dialmoth opened the pull request:
Add note section to clarify that there is a difference between the PowerShell commands which can be used for dedicated SQL pools (formerly SQL DW) and dedicated SQL pool in an Azure Synapse Analytics workspace

dialmoth committed:
Update sql-data-warehouse-reference-powershell-cmdlets.md

WilliamDAssafMSFT commented on articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-reference-powershell-cmdlets.md:
```diff
@@ -12,10 +12,13 @@ ms.reviewer: wiassaf
+> This article applies for standalone dedicated SQL pools (formerly SQL DW) and are not applicable to a dedicated SQL pool created in an Azure Synapse Analytics workspace. There are different PowerShell cmdlets to use for each, for example, use [Suspend-AzSqlDatabase](https://learn.microsoft.com/powershell/module/az.sql/suspend-azsqldatabase) for a dedicated SQL pool (formerly SQL DW), but Suspen ...
+
 [!INCLUDE [updated-for-az](../../../includes/updated-for-az.md)]
 
```
```suggestion
[!INCLUDE [updated-for-az](../../../includes/updated-for-az.md)]

> [!NOTE]
> This article applies for standalone dedicated SQL pools (formerly SQL DW) and are not applicable to a dedicated SQL pool created in an Azure Synapse Analytics workspace. There are different PowerShell cmdlets to use for each, for example, use [Suspend-AzSqlDatabase](/powershell/module/az.sql/suspend-azsqldatabase) for a dedicated SQL pool (formerly SQL DW), but Suspend-[AzSynapseSqlPool](powershell/module/az.synapse/suspend-azsynapsesqlpool) for a dedicated SQL pool in an Azure Synapse Workspace. For instructions to pause and resume a a dedicated SQL pool created in an Azure Synapse Analytics workspace, see [Quickstart: Pause and resume compute in dedicated SQL pool in a Synapse Workspace with Azure PowerShell](pause-and-resume-compute-workspace-powershell.md). For more on the differences between dedicated SQL pool (formerly SQL DW) and dedicated SQL pools in Azure Synapse Workspaces, read [What's the difference between Azure Synapse (formerly SQL DW) and Azure Synapse Analytics Workspace](https://techcommunity.microsoft.com/t5/azure-synapse-analytics-blog/what-s-the-difference-between-azure-synapse-formerly-sql-dw-and/ba-p/3597772).
```

WilliamDAssafMSFT replied:
Fixed links, moved down.

dialmoth replied:
Fixed the link text from "Suspend-[AzSynapseSqlPool]" to "[Suspend-AzSynapseSqlPool]"

dialmoth committed:
Update sql-data-warehouse-reference-powershell-cmdlets.md

Fixed the link text from "Suspend-[AzSynapseSqlPool]" to "[Suspend-AzSynapseSqlPool]"

dialmoth committed:
Update sql-data-warehouse-reference-powershell-cmdlets.md

Modified the main file
Modified the include link location

dialmoth committed:
Update sql-data-warehouse-reference-powershell-cmdlets.md

Modify the links

WilliamDAssafMSFT commented on articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-reference-powershell-cmdlets.md:
```diff
@@ -12,12 +12,15 @@ ms.reviewer: wiassaf
 [!INCLUDE [updated-for-az](../../../includes/updated-for-az.md)]
 
+> [!NOTE]
+> This article applies for standalone dedicated SQL pools (formerly SQL DW) and are not applicable to a dedicated SQL pool created in an Azure Synapse Analytics workspace. There are different PowerShell cmdlets to use for each, for example, use [Suspend-AzSqlDatabase](/powershell/module/az.sql/suspend-azsqldatabase) for a dedicated SQL pool (formerly SQL DW), but [Suspend-AzSynapseSqlPool](/power ...
```
```suggestion
> This article applies for standalone dedicated SQL pools (formerly SQL DW) and are not applicable to a dedicated SQL pool created in an Azure Synapse Analytics workspace. There are different PowerShell cmdlets to use for each, for example, use [Suspend-AzSqlDatabase](/powershell/module/az.sql/suspend-azsqldatabase) for a dedicated SQL pool (formerly SQL DW), but [Suspend-AzSynapseSqlPool](/powershell/module/az.synapse/suspend-azsynapsesqlpool) for a dedicated SQL pool in an Azure Synapse Workspace. For instructions to pause and resume a dedicated SQL pool created in an Azure Synapse Analytics workspace, see [Quickstart: Pause and resume compute in dedicated SQL pool in a Synapse Workspace with Azure PowerShell](pause-and-resume-compute-workspace-powershell.md). For more on the differences between dedicated SQL pool (formerly SQL DW) and dedicated SQL pools in Azure Synapse Workspaces, read [What's the difference between Azure Synapse (formerly SQL DW) and Azure Synapse Analytics Workspace](https://techcommunity.microsoft.com/t5/azure-synapse-analytics-blog/what-s-the-difference-between-azure-synapse-formerly-sql-dw-and/ba-p/3597772).
```

WilliamDAssafMSFT replied:
@dialmoth A couple suggestions. Removing the leading `/` and removing a repeated `a`. Near this comment in GitHub, you should be able to select a button to Commit this suggestion.

dialmoth replied:
Done.
## Removed the leading /
## Removed the repeated a in "resume a a dedicated SQL pool"

dialmoth committed:
Update sql-data-warehouse-reference-powershell-cmdlets.md

## Removed the leading /
## Removed the duplicate a within "resume a a dedicated SQL pool"

Jak-MS committed:
Update sql-data-warehouse-reference-powershell-cmdlets.md

fix links
